crates/common_utils/Cargo.toml

```diff
@@ -47,6 +36,7 @@ md5 = "0.7.0"
 # First party crates
 masking = { version = "0.1.0", path = "../masking" }
 router_env = { version = "0.1.0", path = "../router_env", features = ["log_extra_implicit_fields", "log_custom_entries_to_extra"], optional = true }
+diesel = "2.0.3"
```

Comment:
It'd be great if we could have this behind a `diesel` feature flag.

@Narayanbhat166 @NishantJoshi00 Or would you prefer the type being moved to another crate?